extensions/podman/packages/extension/src/remote/podman-remote-tcp-tunnel.ts

```
@@ -0,0 +1,146 @@
+ ***********************************************************************/
+import * as net from 'node:net';
+
+import type { ProviderConnectionStatus } from '@podman-desktop/api';
+
+export class PodmanRemoteDirectTunnel {
+  // local server for listening on the local file socket
+  #server: net.Server | undefined;
+
+  #status: ProviderConnectionStatus = 'unknown';
+
+  #reconnect: boolean = false;
```

Biome (2.3.13)

[error] This type annotation is trivially inferred from its initialization.

Safe fix: Remove the type annotation.


(lint/style/noInferrableTypes)